oem/ibm/configurations/bios/ibm,bonnell/enum_attrs.json

```diff
@@ -132,15 +132,27 @@
         },
         {
             "attribute_name": "pvm_default_os_type",
-            "possible_values": ["AIX", "Linux", "IBM I", "Linux KVM"],
-            "default_values": ["AIX"],
+            "possible_values": [
+                "AIX",
+                "Linux",
+                "IBM I",
+                "Linux KVM",
+                "Default"
+            ],
+            "default_values": ["Default"],
```

Comment:
Archana, This change would be from Varsha's another PR #493 , could you mention it in the commit message.